packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/angular/load-app.md

```diff
@@ -0,0 +1,80 @@
+The first step for any application is to install or embed the Widget. You can link out to the Okta CDN or install locally through a package manager (for example, `npm`).
+
+### To use the CDN, include the following tags in your HTML:
+
+```html
+<!-- Latest CDN production JavaScript and CSS -->
+<script src="https://global.oktacdn.com/okta-signin-widget/-=OKTA_REPLACE_WITH_WIDGET_VERSION=-/js/okta-sign-in.min.js" type="text/javascript"></script>
+<link href="https://global.oktacdn.com/okta-signin-widget/-=OKTA_REPLACE_WITH_WIDGET_VERSION=-/css/okta-sign-in.min.css" type="text/css" rel="stylesheet"/>
+```
+
+More information, including the latest published version, is available in the [Okta Sign-In Widget Documentation](https://github.com/okta/okta-signin-widget#using-the-okta-cdn).
+
+### npm
+
+```bash
+# Run this command in your project root folder.
+npm install @okta/okta-signin-widget@-=OKTA_REPLACE_WITH_WIDGET_VERSION=-
+```
+
+More information, including the latest published version, is available in the [Okta Sign-In Widget Documentation](https://github.com/okta/okta-signin-widget#using-the-npm-module).
+
+The sample application embeds the Sign-In Widget in the `login.components.ts` file:
+
+```javascript
+import { Component, OnInit } from '@angular/core';
+import { OktaAuth, Tokens } from '@okta/okta-auth-js';
+
+// @ts-ignore
+import * as OktaSignIn from '@okta/okta-signin-widget';
+import sampleConfig from '../app.config';
+
+const DEFAULT_ORIGINAL_URI = window.location.origin;
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent implements OnInit {
+  signIn: any;
+  constructor(public oktaAuth: OktaAuth) {
+    this.signIn = new OktaSignIn({
+      /**
+       * Note: when using the Sign-In Widget for an OIDC flow, it still
+       * needs to be configured with the base URL for your Okta Org. Here
```

Comment:
```suggestion
       * need to configure it with the base URL for your Okta Org. Here
```

Reply:
recast as follows: "you still need to configure the base URL for your Okta Org."